### ISSUE TITLE
Epic: NU6

### ISSUE THREAD

**Issue by mpguerra** [ZcashFoundation/zebra#7957]
## Motivation

Issue to track work required to implement NU6 updates in Zebra.

## Scope

This will be a list of tasks that need to be done for now:

- [x] #7958

Each team can use different activation heights for their private testnets. 

## Before Deploying a Testnet

- Add a NU6 activation height and update the network protocol version

This activates the required NU6 network protocol version, v6 transactions, and anything else that depends on NU6.

**Comment by mpguerra** [comment]
Closing this in favour of the [NU6 Milestone ](https://github.com/ZcashFoundation/zebra/milestone/38)
